docker-compose.yml

```diff
@@ -57,6 +57,13 @@ services:
     depends_on:
       - varnish
 
+  memcached:
+    build:
+      context: ./memcached/1.4/
+    image: quay.io/continuouspipe/memcached1.4:v1.0
```

Comment:
yeah it looks like... don't like this name tho... @andytson-inviqa what's your PoV?